### PR TITLE
Set polling interval to 50ms.

### PR DESCRIPTION
**Pull request by dkocher**

### CODE REVIEW

File: core/src/main/java/ch/cyberduck/core/preferences/Preferences.java [hunk 1]
```diff
@@ -875,7 +875,7 @@ Current default browser view is outline view (0-List view, 1-Outline view, 2-Col
         this.setDefault("sds.upload.sharelinks.keep", String.valueOf(true));
         this.setDefault("sds.upload.s3.enable", String.valueOf(true));
         this.setDefault("sds.versioning.references.enable", String.valueOf(false));
-        this.setDefault("sds.upload.s3.status.period", String.valueOf(1000)); // 1 second
+        this.setDefault("sds.upload.s3.status.period", String.valueOf(100)); // 1ms
```
Comment (AliveDevil) [inline core/src/main/java/ch/cyberduck/core/preferences/Preferences.java:878]: 1ms != 100ms.

**Review by AliveDevil — CHANGES_REQUESTED** [review]

File: core/src/main/java/ch/cyberduck/core/preferences/Preferences.java [hunk 2]
```diff
@@ -875,7 +875,8 @@ Current default browser view is outline view (0-List view, 1-Outline view, 2-Col
         this.setDefault("sds.upload.sharelinks.keep", String.valueOf(true));
         this.setDefault("sds.upload.s3.enable", String.valueOf(true));
         this.setDefault("sds.versioning.references.enable", String.valueOf(false));
-        this.setDefault("sds.upload.s3.status.period", String.valueOf(1000)); // 1 second
+        this.setDefault("sds.upload.s3.status.delay", String.valueOf(5)); // In millis
```
Comment (ylangisc) [inline core/src/main/java/ch/cyberduck/core/preferences/Preferences.java:878]: Should be 50?

Comment (AliveDevil) [inline core/src/main/java/ch/cyberduck/core/preferences/Preferences.java:878]: Review, 5ms aren't 50.

**Review by AliveDevil — CHANGES_REQUESTED** [review]

File: dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java [hunk 3]
```diff
@@ -245,7 +245,8 @@ public void run() {
                         done.countDown();
                     }
                 }
-            }, new HostPreferences(session.getHost()).getLong("sds.upload.s3.status.period"), TimeUnit.MILLISECONDS);
+                                                     }, new HostPreferences(session.getHost()).getLong("sds.upload.s3.status.delay"),
```
Comment (AliveDevil) [inline dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java:248]: Indentation.
Eventually add `delay: ` and `period: ` explicit parameter labels in order to see what each line does.

**Review by AliveDevil — CHANGES_REQUESTED** [review]

File: dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java (hunk 3 above)
Comment (dkocher) [reply dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java:248]: Not aware of such a syntax.

Comment (AliveDevil) [reply dracoon/src/main/java/ch/cyberduck/core/sds/SDSDirectS3UploadFeature.java]: Using `/* delay */` is probably not an option then.

**Review by AliveDevil — APPROVED** [review]

**Pull request merged**